### ISSUE TITLE
Output panel is not shown if execute anonymous code block using Alt + X

### ISSUE THREAD

**Issue by ask9** [dbeaver/dbeaver#34744]
### Description

With Ctrl + Enter it shows Output tab.
With Alt + X it does not show Output tab but I expect it to be shown.
Execute the query on a new tab, copy-pasting the code, to reproduce the issue.
```

do $$ 
begin 
 raise notice 'Notice MESSAGE  %', now();

end $$;

```

On the video, I first use alt + X, after that - ctrl + Enter:

https://github.com/dbeaver/dbeaver/assets/33929221/a5abed0c-6824-4a78-9700-7d19a40629b9

### DBeaver Version

Community Edition 24.1.3.202407042322

### Operating System

Windows 10

### Database and driver

PostgreSQL 16
PostgreSQL JDBC Driver 42.7.2

**Comment by E1izabeth** [comment]
Thank you for bug report

**Comment by ask9** [comment]
@arhayka 
Aren't these two issues duplicates?
If so, why do they have different milestones?

**Comment by E1izabeth** [comment]
@ask9 you're right.
And we fixed this bug in #34865
Here is the Early Access version with fix (https://dbeaver.io/files/ea/)
